sql/server_component/clone_protocol_service.cc

```diff
@@ -307,12 +307,8 @@ DEFINE_METHOD(int, mysql_clone_validate_configs,
     because user can no way clone from the current donor. */
     if (config_name.compare("version_compile_os") == 0) {
       critical_error = ER_CLONE_OS;
-    } else if (config_name.compare("version") == 0) {
-      /* we want to allow to add some suffix to the version and still match
-      i.e. 8.0.25 should be the same as 8.0.25-debug */
-      if (compare_prefix_version(config_val, donor_val)) {
-        continue;
-      }
+    } else if (config_name.compare("version_comment") == 0) {
```

Comment:
since we change the config name here. Do we need to bump the protocol version?

Reply:
I don't think it has any practical difference whether we do or we do not, as the Oracle <-> FB clone will never be supported, and the server version check is already stricter.